Global database and summary sheet.R

```diff
@@ -342,50 +333,246 @@ riskflags <- inner_join(riskflags,
                         keep = F
 )
 
-#-----------------------------Calculate emerging risk score using existing risk ----------------------------------------------
+#-----------------------------—Calculate Overall Alert w/ Geometric Mean ----------------------------------------------
```

Comment:
Important. This, and the following "Calculate Overall Alert w/ ——" sections are code for calculating overall dimensions from existing and emerging risk (now referred to as Underlying Vulnerability and Emerging Threat in documentation).

The sections each follow the same format, using it's specific aggregation method:
1. Calculating the combined metric for each dimension (e.g. calculating overall health from existing and emerging health). These variables are named `OVERALL_[dimension]_[method]`
2. Counting the number of high and medium alert flags, given this method. These variables are named `OVERALL_FLAGS_[method]` when only high risk values are counted (dimension is >= 7) and `OVERALL_FLAGS_[method]_MED` when medium values are also counted (dimension is >= 5 and <7; medium values are counted as 0.5 flags)